module/VuFind/src/VuFind/ILS/Logic/Holds.php

```diff
@@ -83,6 +83,20 @@ class Holds
      */
     protected $hideHoldings = [];
 
+    /**
+     * Search backend
+     *
+     * @var string?
+     */
+    protected $searchBackend = DEFAULT_SEARCH_BACKEND;
+
+    /**
+     * Original Id
+     *
+     * @var string?
+     */
+    protected $originalId = null;
+
```

Comment:
Pretty sure we don't need these any more; I think I just forgot to delete them when I did my refactoring recently...

```suggestion
```

I'm out of time for testing today, so I'm just leaving this as a suggestion -- but if you agree, please accept the suggestion and give this another quick test to be sure we're not missing something.

Reply:
I can confirm that these changes work correctly in my environment. Thank you so much for doing this and providing a foundation on which I hopefully can build...